lib/ebay_request/error_item.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# A single error from many returned in API response
+class EbayRequest::ErrorItem
```

Comment:
Why not use dry-initializer interface here, which is already in the gem's dependencies?

```ruby
class EbayRequest::ErrorItem
  extend Dry::Initializer

  option :errorId,  proc(&:to_s), as: :code
  option :message,  proc(&:to_s)
  option :severity, proc(&:to_s)
  option :params,   proc(&:to_h), default: -> {}

  alias :to_s :message

  def inspect
    "#<#{self.class.name} #{severity} #{code}: #{message.inspect}>"
  end

  def self.new(source)
    # accept source with string keys
    source = source.to_h.each_with_object({}) { |(k, v), obj| obj[k.to_sym] = v }
    super(source)
  end
end
```

Reply:
Oh yes, didn't notice that it is available. Changed to use it.

Decided not to use `errorId` as only half of APIs use this name. Other half uses `errorCode`. I think it is better not to bound naming to any single API